index.js

```diff
@@ -0,0 +1,100 @@
+import addBook from './modules/addBook.js';
+import removeBook from './modules/removeBook.js';
+import { DateTime } from './modules/luxon.js';
+import renderBookList from './modules/renderBookList.js';
+
+class BookList {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+
+    this.errorElements = {
+      author: document.getElementById('authorError'),
+      title: document.getElementById('titleError'),
+    };
+  }
+
+  init() {
+    const navLinks = document.querySelectorAll('nav a');
+    navLinks.forEach((link) => {
+      link.addEventListener('click', (event) => {
+        event.preventDefault();
+
+        navLinks.forEach((navLink) => {
+          navLink.classList.remove('active');
+        });
+
+        link.classList.add('active');
+
+        const sectionId = link.getAttribute('href').substring(1);
+        this.showSection(sectionId);
+      });
+    });
+
+    const addButton = document.getElementById('add-button');
+    addButton.addEventListener('click', (event) => {
+      event.preventDefault();
+      const title = document.getElementById('title').value;
+      const author = document.getElementById('author').value;
+      if (title && author !== '') {
+        const book = { title, author };
+        addBook(book, this.books);
+        renderBookList(this.books, this.removeBook.bind(this));
+
+        document.getElementById('title').value = '';
+        document.getElementById('author').value = '';
+      } else if (title === '') {
+        this.showError('title', 'Title cannot be empty.');
+      } else if (author === '') {
+        this.showError('author', 'Author cannot be empty.');
+      }
+    });
```

Comment:
- [OPTIONAL] After submitting a book, it would be nice to display a message that says the book was added successfully. This will improve the user's interaction with the application. 👍